### ISSUE TITLE
VoiceOver users are unable to click buttons inside swiper pages

### ISSUE THREAD

**Issue by Nigh7Sh4de** [easyui/react-native-ezswiper#8]
Each swiper is currently being wrapped in a `TouchableWithoutFeedback` which as a `Touchable` is set to `accessible={true}` by default. This is preventing VoiceOver users from interacting with `Touchable` elements inside the page.

**Comment by Nigh7Sh4de** [comment]
#7 PR. Please merge this into master and republish to npm.

**Comment by easyui** [comment]
merged,thx
